learn/tanssi/technical.md

```diff
@@ -1,53 +1,95 @@
 
-Tanssi addresses the most common Appchain problems by:
+In the Polkadot ecosystem, the Appchains connected to the relay chain are fully sovereign blockchains, having their own rules, consensus mechanisms, and so forth, and this is the case for the Tanssi network and the Appchains deployed through Tanssi as well. 
 
-- **Block Production as a Service** - Appchains built with Tanssi will get their block produced by Tanssi incentivized workers (block-producers). Tanssi guarantees Appchain's liveliness and a decentralized set of block-producers
+The Tanssi network and the ContainerChains can be considered sibling chains, meaning that there is no hierarchical dependency whatsoever. Their responsibility and how they interact with each other through the relay chain will be covered in the following sections.
 
-- **Consensus on Demand** - Appchains will inherent block finality (consensus) from Polkadot, either on a continous basis ([Parachain](XXX)), or on a pay-as-you-go model ([Parathread](XXX)). Consequently, Appchains built with Tanssi will have access to a robust consensus mechanism from the genesis block
+![Sibling Chains](/images/learn/tanssi/technical/light-technical-2.png#only-light)
+![Sibling Chains](/images/learn/tanssi/technical/dark-technical-2.png#only-dark)
 
-- **Modular Blockchain Framework** - Appchains built with Tanssi can build using a modular blockchain framework (called [Substrate](https://substrate.io/){target=_blank}) that enables developers to quicky and easily build optimized blockcahins for any use case. Tanssi will handle most infrastructural complexities so that developers can focus on their Appchain custom logic
+### Collator Assignment {: #collators-assignment } 
 
-- **Key Integrations** - Appchains built with Tanssi with have access to key infrastuctural components alongisde block production. Because these Appchains are built with [Substrate](https://substrate.io/){target=_blank}, crucial components like wallets, block explorers, indexers, RPC providers, and others are supported out of the box. Furthermore, Appchains can leverage multiple interoperability protocols to connect to other blockchain ecosystems like Ethereum, Avalance, Binance Smart Chain and more
+The Tanssi protocol manages a set of collators and assigns them to provide block production services to the active ContainerChains and the Tanssi network itself.
 
+The assignment algorithm will start distributing the available collators, serving first the Tanssi network and then the ContainerChains, ordered by the registration date, on a first-come, first-served basis. Once the assignment is made, it will be upheld for at least one session, which represents a period measured in blocks that has a constant set of collators. In the provided [templates](/learn/tanssi/templates), the default session duration is set to 1800 blocks, which translates to (roughly) six hours.
```

Comment:
```suggestion
The assignment algorithm will start distributing the available collators, serving first the Tanssi network and then the ContainerChains, ordered by the registration date, on a first-come, first-served basis. Once the assignment is made, it will be upheld for at least one session, which represents a period measured in blocks that has a constant set of collators. In the provided [templates](/learn/tanssi/templates), the default session duration is set to 1800 blocks, which  with an average block time of 12 seconds, it translates to (roughly) six hours.
```